src/components/deltabargraph.js

```diff
@@ -18,8 +23,15 @@ function DeltaBarGraph({timeseries, arrayKey}) {
 
     const margin = {top: 50, right: 0, bottom: 50, left: 0};
     const chartRight = width - margin.right;
-    const chartBottom = height - margin.bottom;
+    let chartBottom = height - margin.bottom;
     const barRadius = 5;
+    if (arrayKey === 'dailyactive') {
+      for (const day of data) {
+        if (day[arrayKey] < 0) {
+          chartBottom = height - margin.bottom * 2;
+        }
+      }
+    }
```

Comment:
Why make the active chart smaller?

Reply:
since, active chart contain both -ve and +ve values. so, i did this otherwise it will look like this. few values will cut down.
![Screenshot from 2020-04-28 18-23-20](https://user-images.githubusercontent.com/45959932/80794904-b2e7cb00-8bb8-11ea-9d9c-fbd7122b8d85.png)

Reply:
I think tweaking the domain should fix that

Reply:
i tried to fix it by tweaking the domain. i'm not able to do it... i think without changing chartBottom it's not possible to fit -ve values.

Reply:
active delta bar graph
![Screenshot from 2020-05-01 20-40-52](https://user-images.githubusercontent.com/45959932/80815904-17704d80-8bec-11ea-9626-fe7bf7fdb64a.png)